fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/base/InsertionOnlyStorage.java

```diff
@@ -38,4 +42,9 @@ default boolean supportsExtraction() {
 	default long extract(T resource, long maxAmount, TransactionContext transaction) {
 		return 0;
 	}
+
+	@Override
+	default Iterator<StorageView<T>> iterator() {
```

Comment:
The overrides in `TrashingStorage` (in testmod) and `TopStorage` (in `ComposterWrapper`) can now be removed.